### DISCUSSION TITLE
Install Yeoman VS Code extension inside Codespace

### DISCUSSION THREAD

**Discussion by ajskateboarder** [community/community#32789] [Codespaces] [answered]
Hello, Codespaces community,

I'm trying to create a Language Support extension inside of a codespace. The quickstart for a new extension says to install it by moving the extension directory to `$HOME/.vscode/extensions`, although that folder doesn't even exist. I've tried making the directory and adding it, but the extension does not apply. Adding the .vscode dir to the actual project doesn't work either.

Can anyone help?

Thanks

**Comment by dmilburn** [comment]
Hi @themysticsavages, thanks for reaching out. I just tested out the instructions found at https://code.visualstudio.com/api/get-started/your-first-extension in a codespace and was able to get their hello world extension running in a new window after running F5. As a first debugging step, would you let me know if those instructions work for you on the hello world extension?

**Reply by ajskateboarder** [reply]
Hello,
I went to the link you mentioned and they do mention pressing F5 to start a new **Extension Development Host**. Since I don't have function keys that map to the shortcut in the `launch.json` on my laptop, I manually started code with the flags in the launch.json:
```bash
code . --extensionDevelopmentPath=.
``` 
However, I noticed a log when code started in a new tab:
```
Ignoring option 'extensionDevelopmentPath': not supported for code.
```
And here's my launch.json for reference:
```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Extension",
            "type": "extensionHost",
            "request": "launch",
            "args": [
                "--extensionDevelopmentPath=${workspaceFolder}"
            ]
        }
    ]
}
```

Thanks

**Reply by ajskateboarder** [reply]
Also, I mean Language Support extensions and not real JS/TS extensions

**Comment by ajskateboarder** [comment, marked as answer]
Guess I solved my own answer. Codespaces use the `.vscode-remote` directory instead of `.vscode` for storing extensions and global configs. Moving my language support extension to `.vscode-remote/extensions` brought syntax highlighting.
